app.rb

```diff
@@ -0,0 +1,56 @@
+require './classroom'
+require './rental'
+require './student'
```

Comment:
- [ ] Please  you will have to import the `person.rb` file into the `student.rb` file  since the `Student class` inherits from the `Person class`. If don't import it, you app will not run. Something like so: 
```
require_relative './person'
```
This should be added  to the `student.rb` file.